### ISSUE TITLE
A question about "Loss is nan"

### ISSUE THREAD

**Issue by yafangna** [vtddggg/Robust-Vision-Transformer#7]
Is there any interface in the source code that would cause the loss is nan problem sometimes and the correct operation sometimes?

**Comment by vtddggg** [comment]
"loss is nan" is a problem for original ViT models when amp is turned on. Check [here](https://github.com/facebookresearch/deit/issues/29) for more details. 

Afterwards, many techniques are proposed to solve this problem, e.g., LayerScale. You can refer these techniques for preventing training loss to nan.
